### PR TITLE
Refs [TASK][34646] Create UI Favorite Cities Screen

### PR DESCRIPTION
**Pull request by I-am-Lien**
## Related Tickets
- [#Task 34646:  Create UI Favorite Cities Screen](https://edu-redmine.sun-asterisk.vn/issues/34646)
## WHAT
- Create UI Favorite Cities Screen
## Evidence (Screenshot or Video)
favio

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/item_favorite.xml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        app:layout_constraintEnd_toEndOf="@+id/imageWindSpeedFavorite"
+        app:layout_constraintStart_toStartOf="@+id/imageWindSpeedFavorite"
+        app:layout_constraintTop_toBottomOf="@id/imageWindSpeedFavorite"
+        app:tint="?attr/color_tint_image_view" />
+
+    <TextView
+        android:id="@+id/textWindSpeedFavorite"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="8dp"
```
Comment (Domhoan) [inline app/src/main/res/layout/item_favorite.xml]: còn chỗ này bạn ơi

Comment (thonc-2135) [reply app/src/main/res/layout/item_favorite.xml]: tưởng kêu ngủ mà chăm thế =)

File: app/src/main/res/layout/item_favorite.xml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        app:tint="?attr/color_tint_image_view" />
+
+    <ImageView
+        android:id="@+id/imageHumidityFavorite"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/dp_12"
+        android:src="@drawable/ic_humidity"
+        app:layout_constraintEnd_toEndOf="@+id/imageWindSpeedFavorite"
+        app:layout_constraintStart_toStartOf="@+id/imageWindSpeedFavorite"
```
Comment (Domhoan) [inline app/src/main/res/layout/item_favorite.xml]:
```suggestion
        app:layout_constraintEnd_toStartOf="@id/textWindSpeedFavorite"
        app:layout_constraintStart_toEndOf="@id/textDegreeFavorite"
```
suggest những chỗ constraint để như này

**Review by Domhoan — COMMENTED** [review]
Tên ticket chưa sửa kìa Liên

**Pull request merged**